src/main/java/app/knock/api/model/KnockTenant.java

```diff
@@ -0,0 +1,40 @@
+package app.knock.api.model;
+
+import com.fasterxml.jackson.annotation.JsonAnySetter;
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.Builder;
+import lombok.Singular;
+import lombok.Value;
+import lombok.extern.jackson.Jacksonized;
+
+import java.time.ZonedDateTime;
+import java.util.Map;
+
+@Value
+@Jacksonized
+@Builder
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class KnockTenant {
+
+    @JsonProperty("__typename")
+    String typeName;
+
+    String id;
+
+    ZonedDateTime createdAt;
+    ZonedDateTime updatedAt;
+
+    @Singular("data")
```

Comment:
This should be pointing at `properties` – the value here is what's coming back in the JSON response